OpenRA.sln

```diff
@@ -72,11 +72,4 @@ Global
 	GlobalSection(ExtensibilityGlobals) = postSolution
 		SolutionGuid = {9E2FAACA-DADB-4EEB-9898-2B0269F45E69}
 	EndGlobalSection
-	GlobalSection(MonoDevelopProperties) = preSolution
```

Comment:
Changes related to the Monodevelop IDE can stay.

Reply:
https://github.com/mono/monodevelop is dead too.

Reply:
Likewise here.